src/phoenix/trace/trace_dataset.py

```diff
@@ -37,6 +54,38 @@ def normalize_dataframe(dataframe: DataFrame) -> "DataFrame":
     return dataframe
 
 
+def _delete_empty_document_metadata(documents: Any) -> Any:
+    """
+    Adds a dummy field to the document metadata to make the object serializable over parquet
+    """
+    # If the documents is a list, iterate over them, check that the metadata is
+    # a dict, see if it is empty, and if it's empty, delete the metadata
+    if isinstance(documents, list):
+        # Make a deep copy of the documents
+        documents = copy.deepcopy(documents)
```

Comment:
Here's a shallower alternative: copying just the keys. It probably won't make any real difference, but does avoids copying the dict that we're going to delete. (See variable `z` in screenshot below.)
```suggestion
        documents = list(map(dict, documents)))
```
<img width="286" alt="Screenshot 2023-12-01 at 5 19 48 PM" src="https://github.com/Arize-ai/phoenix/assets/80478925/cd7b38c3-443a-42ee-a0b8-3d3af6b94efc">

Reply:
Ah man, so many pointers. Gotcha.